### ISSUE TITLE
Write additional tests

### ISSUE THREAD

**Issue by willzeng** [willzeng/cograph#66]
Perhaps using jasmine:
http://jasmine.github.io/

**Comment by harlantwood** [comment]
I've been very happy with mocha + chai for testing server side (nodejs) code.  Here's an example from my latest project of how the tests look: https://github.com/nodesphere/nodesphere/blob/master/test/core/node_tests.coffee

Does this sounds interesting, or do you prefer Jasmine?

**Comment by harlantwood** [comment]
Ah, I found your tests in https://github.com/willzeng/cograph/tree/dev/app/tests -- are these working for you?  If so, maybe close this issue?  If not, why not?

**Comment by davidfurlong** [comment]
The test setup is working, but we haven't written ones that we need in order to increase the reliability of a build. This issue is more about writing particular tests than setting up the test environment.

**Comment by willzeng** [comment]
That said if you have suggestions for better ways to set the tests up they
would be welcome!   Now would be the time to switch away from Jasmine
before tests are written.

**Comment by harlantwood** [comment]
Cool.  I got the current tests to run by doing:

```
http-server
open http://localhost:8080/app/tests/SpecRunner.html
```

Is there a way to run them from the command line as well?

**Comment by willzeng** [comment]
Yep spot on that's it. I'm not sure of any way to see them in the command
line as Jasmine really does seem to run them in the browser.  Would be an
improvement if there was a way.
